dasharo-security/me-keyword.resource

```diff
@@ -0,0 +1,9 @@
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check If ME Is Disabled
+    ${out}=    Run Process    /usr/sbin/setpci    -s    16.0    40.L
+    RETURN    Should Not Be Equal As Strings    ${out.stdout}[3]    0
```

Comment:
Adding some comment here would be useful - e.g. we check for bit number X which should inidcate something

Reply:
Is this return value useful ? Do we rely on it somehow? I see the kwd is called, but we do not assign return value anywhere.

Reply:
now it has a use, everything is explained in comment which i also added